tests/flags/quiet.rs

```diff
@@ -20,7 +20,7 @@ fn fix_output_in_quiet_mode() {
     let _ = test_dir.create_testfile(".env", "abc=DEF\n\nF=BAR\nB=bbb\n");
 
     let args = &["--quiet"];
-    let expected_output = format!("All warnings are fixed. Total: {}\n", 2);
+    let expected_output = format!("\nAll warnings are fixed. Total: {}\n", 2);
```

Comment:
Is that necessary?

I got the result with 2 blank lines before this message:
```
Checking .env
Original file was backed up to: ".env_1602361743"

.env:4 DuplicatedKey: The A2 key is duplicated
.env:4 LeadingCharacter: Invalid leading character detected


All warnings are fixed. Total: 2
```

Reply:
Ah, good catch.